internal/app/kdk/init.go

```diff
@@ -42,6 +46,12 @@ func InitKdkConfig(logger logrus.Entry) error {
 
 	currentUser, _ := user.Current()
 
+	username := currentUser.Username
+
```

Comment:
Please add a comment.  I assume that the username has \\ in it because of Windows?

Reply:
comment added